llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2264,7 +2265,7 @@ class VPWidenIntOrFpInductionRecipe : public VPWidenInductionRecipe,
   /// Returns the scalar type of the induction.
   Type *getScalarType() const {
     return Trunc ? Trunc->getType()
-                 : getStartValue()->getLiveInIRValue()->getType();
+                 : cast<VPLiveIn>(getStartValue())->getValue()->getType();
```

Comment:
```suggestion
                 : cast<VPLiveIn>(getStartValue())->getType();
```
Should getStartValue return a VPLiveIn?